tests/specs/cli/otel_basic/http_propagators.out

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
Because of `DENO_UNSTABLE_OTEL_DETERMINISTIC`, there are now multiple span ids named `0000000000000001` here - that's not good.

We need to update `DENO_UNSTABLE_OTEL_DETERMINISTIC` to take a prefix to use for span IDs, so that different processes generate different (sequential) span IDs. I'll push a fix for this shortly.